cinema/serializers.py

```diff
@@ -130,13 +130,17 @@ class Meta:
         model = Order
         fields = ("id", "tickets", "created_at",)
 
+    @transaction.atomic()
```

Comment:
you don't need parentheses because you don't pass any parameters to this decorator